### PR TITLE
BUGFIX: CL-9654 Added measure index mapping to response headers

### PR DESCRIPTION
**Pull request by bredy**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded.

- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul/191/ : SUCCESS in 2m 11s

**Comment by yenkins-admin** [comment]
Merged build triggered.

File: test/execution_test.js [hunk 1]
```diff
@@ -154,6 +154,41 @@ describe('execution', () => {
                         done();
                     });
                 });
+
+                it('should wrap response headers with metric mappings', done => {
```
Comment (akloboucnik) [inline test/execution_test.js]: remove `done`, return promise from test fn

**Comment by gdgate** [comment]
Build succeeded.

- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul/194/ : SUCCESS in 2m 02s

**Pull request merged**